src/data/aboutYouFormOptions.js

```diff
@@ -0,0 +1,205 @@
+	},
+	{
+		testId: 'asylee',
+		formatMessageId: 'aboutyou.immigration-answer-asylee'
+	},
+	{
+		testId: 'dreamer',
+		formatMessageId: 'aboutyou.immigration-answer-dreamer'
+	},
+	{
+		testId: 'refugee',
+		formatMessageId: 'aboutyou.immigration-answer-refugee'
+	},
+	{
+		testId: 'immigrant',
+		formatMessageId: 'aboutyou.immigration-answer-immigrant'
+	},
+	{
+		testId: 'none',
+		formatMessageId: 'aboutyou.answer-none-apply'
+	},
+	{
+		testId: 'prefer',
+		formatMessageId: 'aboutyou.answer-prefer-not-to-say'
+	}
+];
+
+export const aboutYouCountryOptions = [
+	{
+		testId: 'africa',
+		formatMessageId: 'aboutyou.country-answer-africa'
+	},
+	{
+		testId: 'asia',
+		formatMessageId: 'aboutyou.country-answer-asia'
+	},
+	{
+		testId: 'south-america',
+		formatMessageId: 'aboutyou.country-answer-south-america'
+	},
+	{
+		testId: 'caribbean',
+		formatMessageId: 'aboutyou.country-answer-caribbean'
+	},
+	{
+		testId: 'north-america',
+		formatMessageId: 'aboutyou.country-answer-north-america'
+	},
+	{
+		testId: 'central-america',
+		formatMessageId: 'aboutyou.country-answer-central-america'
+	},
+	{
+		tetId: 'middle-east',
+		formatMessageId: 'aboutyou.country-answer-middle-east'
+	},
+	{
+		testId: 'europe',
+		formatMessageId: 'aboutyou.country-answer-europe'
+	},
+	{
+		testId: 'prefer',
+		formatMessageId: 'aboutyou.answer-prefer-not-to-say'
+	},
+	{
+		testId: 'other',
+		formatMessageId: 'aboutyou.answer-other'
+	}
+];
+
+export const aboutYouSogOptions = [
+	{
+		testId: 'lesbian',
+		formatMessageId: 'aboutyou.identity-answer-lesbian'
+	},
+	{
+		testId: 'gay',
+		formatMessageId: 'aboutyou.identity-answer-gay'
+	},
+	{
+		testId: 'bisexual',
+		formatMessageId: 'aboutyou.identity-answer-bisexual'
+	},
+	{
+		testId: 'transgender',
+		formatMessageId: 'aboutyou.identity-answer-transgender'
+	},
+	{
+		testId: 'queer',
+		formatMessageId: 'aboutyou.identity-answer-queer'
+	},
+	{
+		testId: 'questioning',
+		formatMessageId: 'aboutyou.identity-answer-questioning'
+	},
+	{
+		testId: 'intersex',
+		formatMessageId: 'aboutyou.identity-answer-intersex'
+	},
+	{
+		testId: 'asexual',
+		formatMessageId: 'aboutyou.identity-answer-asexual'
+	},
+	{
+		testId: 'pansexual',
+		formatMessageId: 'aboutyou.identity-answer-pansexual'
+	},
+	{
+		testId: 'nonbinary',
+		formatMessageId: 'aboutyou.identity-answer-nonbinary'
+	},
+	{
+		testId: 'ally',
+		formatMessageId: 'aboutyou.identity-answer-ally'
+	},
+	{
+		testId: 'straight',
+		formatMessageId: 'aboutyou.identity-answer-straight'
+	},
+	{
+		testId: 'woman',
+		formatMessageId: 'aboutyou.identity-answer-woman'
+	},
+	{
+		testId: 'man',
+		formatMessageId: 'aboutyou.identity-answer-man'
+	},
+	{
+		testId: 'prefer',
+		formatMessageId: 'aboutyou.answer-prefer-not-to-say'
+	},
+	{
+		testId: 'other',
+		formatMessageId: 'aboutyou.answer-other'
+	}
+];
+
+export const aboutYouEthnicityOptions = [
+	{
+		testId: 'af-md',
+		formatMessageId: 'aboutyou.ehnicity-answer-af-md'
```

Comment:
"ethnicity" is misspelt in all options

```suggestion
		formatMessageId: 'aboutyou.ethnicity-answer-af-md'
```